crypto/rand_test.go

```diff
@@ -11,6 +11,14 @@ func TestRandIntnPanics(t *testing.T) {
 			t.Error("expected panic for n <= 0")
 		}
 	}()
-	RandIntn(0)
-	RandIntn(-1)
+
+	_, err := RandIntn(0)
+	if err != nil {
+		t.Error("expected panic on n <= 0, not error")
+	}
+
+	_, err = RandIntn(-1)
+	if err != nil {
+		t.Error("expected panic on n <= 0, not error")
+	}
 }
```

Comment:
yeah, `panic` semantics make this slightly annoying to test. You could use a helper function:
```go
func panics(fn func()) (panicked bool) {
    defer func() {
        panicked = (recover() != nil)
    }()
    fn()
    return
}
```
and then test with:
```go
if !panics(func() { RandIntn(0) }) {
    t.Error("RandIntn did not panic")
}
```